peps/pep-0790.rst

```diff
@@ -0,0 +1,469 @@
+PEP: 790
+Title: New environment markers for modern Python environments
```

Comment:
Minor: "modern" always ages poorly, as this PEP will still exist in twenty years yet will certainly not be modern any more. Is there a better term we can use?

Reply:
So true! Done.